d2renderers/d2svg/style.css

```diff
@@ -10,3 +10,20 @@
   mix-blend-mode: multiply;
   opacity: 0.5;
 }
+
+.sketch-overlay-bright {
```

Comment:
i think these will actually still be rare. can you add these conditionally, like https://github.com/terrastruct/d2/blob/ea5db1d424e0d1c635ff1a799d4c8172f86660f1/d2renderers/d2svg/d2svg.go#L1053